src/api/v2/order.rs

```diff
@@ -535,6 +535,17 @@ pub struct ChangeReq {
 }
 
 
+fn empty_to_simple<'de, D>(deserializer: D) -> Result<Class, D::Error>
+where
+    D: Deserializer<'de>,
+{
+    let class = Class::deserialize(deserializer);
+    match class {
+      Ok(value) => Ok(value),
+      Err(_) => Ok(Class::Simple),
```

Comment:
The one issue here is that we silently swallow all kinds of errors. We shouldn't be doing that, or we will mask other legitimate errors. But I already found a way to fix that.